### ISSUE TITLE
treatment with cardiovascular drug use measurement

### ISSUE THREAD

**Issue by gkos-bio** [EBISPOT/efo#932]
Related to the ticket on Lipid-lowering, there is no classification of cardiovascular agent therapy measurement in EFO.
I think this will be extremely beneficial to re organise the drug class use in EFO especially from a GWAS Biobank point of view.

Lipid-lowering drug use measurement could be a child of this one.
"Vasodilators used in cardiac diseases use measurement", "Calcium channel blocker use measurement" will be children of this class as well as others.

Please, see classification of drug therapy in https://www.ebi.ac.uk/ols/ontologies/maxo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMAXO_0000181 for corresponding subclasses some of which are already represented in EFO.

##Preferred term
cardiovascular drug treatment use measurement.

##Synonyms
Can be based on the MAXO synonyms for this class of drugs.

**Comment by gkos-bio** [comment]
Do I need to open another ticket for respiratory system related drug use?

**Comment by zoependlington** [comment]
@gkos-bio Respiratory system related drug use can be added to this ticket too, that's fine.

**Comment by gkos-bio** [comment]
Thank you. I think in general, this branch of EFO could be reclassified following the drug class hierarchy in MAXO if it makes sense.

**Comment by jamesamcl** [comment]
**Status triage — still open after ~5.6 years, never actioned**

**The ask:** add a `cardiovascular drug use measurement` grouping class under `drug use measurement` (EFO:0007010), reparent the existing cardiovascular drug-class terms beneath it, plus (per @zoependlington's reply) an equivalent respiratory grouping; with a longer-term suggestion to realign the branch on the MAXO drug-class hierarchy.

**Current situation:**

- **Not implemented.** No `cardiovascular drug use measurement` term exists in `efo-edit.owl`, and no PR has ever referenced this issue. Activity since Jan 2021 is limited to an unassignment (Sep 2021) and a labelling/board pass (May 2025).
- `drug use measurement` (EFO:0007010) still has **31 direct children in a completely flat list** — no grouping by body system or drug class, and none of the children carries an `is_about` axiom (the parent itself has `is_about some 'drug' (CHEBI:23888)`).
- The intended children all exist and are active: EFO:0009925 Antithrombotic agent, EFO:0009926 Vasodilators used in cardiac diseases, EFO:0009927 Antihypertensive, EFO:0009928 Diuretic, EFO:0009929 Beta blocking agent, EFO:0009930 Calcium channel blocker, EFO:0009931 Agents acting on the renin-angiotensin system, EFO:0009932 HMG CoA reductase inhibitor, and EFO:0803367 antihyperlipidemic drug use measurement (which carries "lipid-lowering drug use measurement" as an exact synonym — the subject of #931).
- **The OBA measurement migration (epic #787, closed 2026-08-04) did not affect this branch.** No `* use measurement` term was obsoleted, which is correct — these are drug-exposure/treatment-history measurements, not biological attributes, so OBA does not supersede them. This issue is still an in-EFO job.

**What makes this cheap now:** `CHEBI:35554` "cardiovascular drug" (synonym "cardiovascular agent") is **already imported** into EFO and is asserted `SubClassOf CHEBI:23888` — the exact filler EFO:0007010 already uses. So the new term drops straight into the existing pattern with no import work:

- **Label:** `cardiovascular drug use measurement` (the flat-noun form matches the branch's convention better than the title's "treatment with…"; "cardiovascular agent use measurement" would make a good related synonym, following CHEBI)
- **Parent:** `drug use measurement` (EFO:0007010)
- **Logical definition:** `is_about some 'cardiovascular drug' (CHEBI:35554)`
- **Then** assert the nine terms above as children.

One caveat worth recording: CHEBI's own `cardiovascular drug` branch is incomplete — only antihypertensive agent (CHEBI:35674) and vasodilator agent (CHEBI:35620) are descendants, while diuretic (CHEBI:35498), calcium channel blocker (CHEBI:38215), beta-adrenergic antagonist (CHEBI:35530) and antilipemic drug (CHEBI:35679) are not. A reasoner working off CHEBI alone would under-classify, so the `is_a` links need to be asserted rather than inferred.

**Suggested scoping — this ticket carries three asks of very different sizes:**

1. **The cardiovascular grouping (this issue).** Small, unblocked, ~1 new term + 9 reparentings. Purely additive, so the live GWAS annotations on EFO:0009925/0009927/0009930 are unaffected. Recommend keeping #932 open for exactly this, retitled `cardiovascular drug use measurement`.
2. **The respiratory grouping.** Recommend splitting to its own issue. There is no CHEBI role for respiratory-system drugs to hang it on (only `bronchodilator agent` CHEBI:35523 and `anti-asthmatic agent` CHEBI:65023), and EFO currently has only EFO:0009941 Inhalant adrenergic and, arguably, EFO:0009943 Antihistamine to put under it — so it needs an ATC-R- or MAXO-based decision that the cardiovascular case does not.
3. **Reclassifying the branch on MAXO** (@gkos-bio's 2021-01-13 comment). MAXO:0000181 `cardiovascular agent therapy` maps well onto this set, but **MAXO is not imported into EFO** (no `maxo_terms.txt`, no MAXO import product). That work belongs to #879 "Make EFO interoperable with MAXO?", which is also still open. Recommend tracking it there rather than here.

One open question for maintainers: is the CHEBI-based grouping in (1) right to action as-is, or would you rather wait on the MAXO decision in #879 so the branch is only restructured once?

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
